aphrodite/modeling/sampling_metadata.py

```diff
@@ -728,6 +747,31 @@ def from_lists(cls, temperatures: List[float], dynatemp_mins: List[float],
                                  device="cpu",
                                  dtype=dtype,
                                  pin_memory=pin_memory)
+        dry_multipliers_t = torch.tensor(
+            dry_multipliers,
+            device="cpu", 
+            dtype=dtype,
+            pin_memory=pin_memory,
+        )
+        dry_bases_t = torch.tensor(
+            dry_bases,
+            device="cpu",
+            dtype=dtype, 
+            pin_memory=pin_memory,
+        )
+        dry_allowed_lengths_t = torch.tensor(
+            dry_allowed_lengths,
+            device="cpu",
+            dtype=torch.int,
+            pin_memory=pin_memory,
+        )
+        dry_sequence_breakers_t = torch.tensor(
```

Comment:
Currently crashes at this line in some cases, I'm likely incorrectly treating sequence breaker array the same as the scalars above here